backend/mora/common.py

```diff
@@ -52,18 +53,22 @@ def get(self, objid):
 
 
 class RequestType(enum.Enum):
-    CREATE = 0
-    EDIT = 1
+    CREATE, EDIT, TERMINATE = range(3)
 
 
 # The handler mapping is populated by each individual active RequestHandler
 HANDLERS = {}
+HANDLERS_BY_KEY = {}
+FUNCTION_KEYS = {}
 
 
-def register_request_handler(name):
+def register_request_handler(name, key=None):
```

Comment:
It feels wrong to add an additional parameter here, that only makes sense for org funk handlers. We have a class that every org funk handler inherits from, so maybe the logic regarding function keys could be moved there instead?